Makefile

```diff
@@ -92,7 +92,12 @@ license:
 	GO111MODULE=off go get -u github.com/google/addlicense && \
 		addlicense -c "Red Hat, Inc" -l "apache" -v ./
 
-before_commit: style test integration_tests openapi-check license
+before_commit: style
+	./unit-tests.sh
+	./unit-tests.sh postgres
+	make integration_tests
+	make openapi-check
+	make license
```

Comment:
Why changing "rule dependencies" to running the scripts? The only advantage I can see right now is that it won't build the application twice, but it can be enhanced in the `build` rule instead of this

Reply:
please rebase, there are some changes made in the same line.
+ would be nice to know answer to @joselsegura question

Reply:
Because I'm dumb :D Fixed.

Reply:
I don't think that you are :-D

TYVM